web/core/src/jsMain/kotlin/org/jetbrains/compose/web/elements/Elements.kt

```diff
@@ -650,22 +650,51 @@ fun Section(
     content = content
 )
 
+/**
+ * Adds <textarea> element.
+ * Same as [Input], [TextArea] has two modes: controlled and uncontrolled.
+ *
+ * Controlled mode means that <textarea> value can be changed only by passing a different [value].
+ * Uncontrolled mode means that <textarea> uses its default state management.
+ *
+ * To use controlled mode, simply pass non-null [value].
+ * By default [value] is null and [TextArea] will be in uncontrolled mode.
+ *
+ * Use `defaultValue("some default text")` in uncontrolled mode to set a default text if needed:
+ *
+ * ```
+ * TextArea {
+ *      defaultValue("Some Default Text")
+ * }
+ * ```
+ */
 @Composable
 fun TextArea(
-    attrs: (TextAreaAttrsBuilder.() -> Unit)? = null,
-    value: String
-) = TagElement(
-    elementBuilder = TextArea,
-    applyAttrs = {
-        val  taab = TextAreaAttrsBuilder()
```

Comment:
**(I do realize that this is the removed code, I just want to use it as a chance to discuss this)**
honestly, let's not use "c-like" variable names,
1) we are using  IDE
2) the benefits of immediately understanding what this variable is about over-weighs the perks of short name)

Reply:
agree